testutil/dbconfig/config_fullpgdb.go

```diff
@@ -0,0 +1,12 @@
+// +build fullpgdb
+
+package dbconfig
+
+// Test DB server and database config.
+const (
+	PGTestsHost   = "localhost" // "/run/postgresql" for UNIX socket
+	PGTestsPort   = "5432"      // "" for UNIX socket
+	PGTestsUser   = "postgres"  // "dcrdata" for full database rather than test data repo
+	PGTestsPass   = ""
+	PGTestsDBName = "dcrdata_mainnet_full"
```

Comment:
The DB config you get with `-tags fullpgdb` will use a different database.  But because of the `dcrpg` replace statement for dbconfig, any of these settings can be modified.